### DISCUSSION TITLE
multiple Operations, Annotation, Basic standard png

### DISCUSSION THREAD

**Discussion by wodan66** [Stirling-Tools/Stirling-PDF#426] [Ideas]
I recently scan some invoices and would like to suggest the following feature:

A) apply some basic "stamp", e.g. (BEZAHLT, PAID, ABGELEHNT, APPROVED, GENEHMIGT). Ideally a transparent PNG.
B) apply a text, e.g. todays date along with my initials or name.
C) Save the PDF

In general I suggest to allow multiple operations. So at a first step you choose the PDF for upload, then you add the 1st manipulation (e.g. delete or rotate pages) then e.g. apply page numbers, then add grafics (or stamp) and finally save the new file as PDF, PDF/A, Image...

I know you already thought about annotations, but probably integration of "workflows" might be a further addon.

**Comment by Serhioromano** [comment]
You can use insert image feature and insert it.
